api/views/session_views.py

```diff
@@ -375,3 +375,24 @@ def get(self, request, summary_id):
         summary = LectureSummary.objects.get(id=summary_id)
         serializer = LectureSummarySerializer(summary)
         return Response(serializer.data, status=status.HTTP_200_OK)
+
+
+@extend_schema(tags=["Summaries"])
+class UpdateLectureSummaryView(APIView):
+    permission_classes = [IsSummaryOwner]
+
+    @extend_schema(
+        request=LectureSummarySerializer,
+        description="Endpoint to update the summary of a lecture given summary_id",
+    )
```

Comment:
It would be good for us to have a serializer for the response here; we can talk about this during tomorrow's/Wednesday's meeting.